docs/manual_tests/user_refresh_token.md

```diff
@@ -0,0 +1,46 @@
+## Manual testing for POST /user/refresh API
+
+#### Dummy data used :
+**username** : testrefresh, 
+**password** : testrefresh@1234
+
+* _Refresh token in Authorization field is the refresh token returned on login response_
```

Comment:
```suggestion
* _Refresh token using the refresh token returned on login response_
```
I changed like this, because the sentence seemed a bit confusing when reading